samples/bluetooth/throughput/src/cmds.c

```diff
@@ -0,0 +1,254 @@
+	struct bt_le_conn_param *conn_param;
+	struct bt_conn_le_phy_param *phy;
+	struct bt_conn_le_data_len_param *data_len;
+} test_params = {
+	.conn_param = BT_LE_CONN_PARAM(0x140, 0x140, 0, 400),
+	.phy = BT_CONN_LE_PHY_PARAM_2M,
+	.data_len = BT_LE_DATA_LEN_PARAM_MAX
+};
+
+extern int test_run(const struct shell *shell,
+		    const struct bt_le_conn_param *conn_param,
+		    const struct bt_conn_le_phy_param *phy,
+		    const struct bt_conn_le_data_len_param *data_len);
+
+static const char *phy_str(const struct bt_conn_le_phy_param *phy)
+{
+	static const char *const str[] = {
+		"1 Mbps",
+		"2 Mbps",
+		"Coded S2",
+		"Coded S8",
+		"Unknown"
+	};
+
+	switch (phy->pref_tx_phy) {
+	case BT_GAP_LE_PHY_1M:
+		return str[0];
+
+	case BT_GAP_LE_PHY_2M:
+		return str[1];
+
+	case BT_GAP_LE_PHY_CODED:
+		if (phy->options == BT_CONN_LE_PHY_OPT_CODED_S2) {
+			return str[2];
+		} else if (phy->options == BT_CONN_LE_PHY_OPT_CODED_S8) {
+			return str[3];
+		}
+
+	default:
+		return str[4];
+	}
+}
+
+static int default_cmd(const struct shell *shell, size_t argc,
+		       char **argv)
+{
+	if (argc == 1) {
+		shell_help(shell);
+		return SHELL_CMD_HELP_PRINTED;
+	}
+
+	if (argc > 2) {
+		shell_error(shell, "%s: bad parameters count.", argv[0]);
+		return -EINVAL;
+	}
+
+	if (argc == 2) {
+		shell_error(shell, "Uknown argument: %s", argv[1]);
+		return -EINVAL;
+	}
+
+	return 0;
+}
+
+static int cmd_phy_1m(const struct shell *shell, size_t argc,
+		      char **argv)
+{
+	test_params.phy->options = BT_CONN_LE_PHY_OPT_NONE;
+	test_params.phy->pref_rx_phy = BT_GAP_LE_PHY_1M;
+	test_params.phy->pref_tx_phy = BT_GAP_LE_PHY_1M;
+
+	shell_print(shell, "PHY set to: %s", phy_str(test_params.phy));
+
+	return 0;
+}
+
+static int cmd_phy_2m(const struct shell *shell, size_t argc,
+		      char **argv)
+{
+	test_params.phy->options = BT_CONN_LE_PHY_OPT_NONE;
+	test_params.phy->pref_rx_phy = BT_GAP_LE_PHY_2M;
+	test_params.phy->pref_tx_phy = BT_GAP_LE_PHY_2M;
+
+	shell_print(shell, "PHY set to: %s", phy_str(test_params.phy));
+
+	return 0;
+}
+
+#if defined(RADIO_MODE_MODE_Ble_LR125Kbit) || defined(NRF5340_XXAA_APPLICATION)
+static int cmd_phy_coded_s2(const struct shell *shell, size_t argc,
+			    char **argv)
+{
+	test_params.phy->options = BT_CONN_LE_PHY_OPT_CODED_S2;
+	test_params.phy->pref_rx_phy = BT_GAP_LE_PHY_CODED;
+	test_params.phy->pref_tx_phy = BT_GAP_LE_PHY_CODED;
+
+	shell_print(shell, "PHY set to: %s", phy_str(test_params.phy));
+
+	return 0;
+}
+#endif /* defined(RADIO_MODE_MODE_Ble_LR125Kbit) ||
+	* defined(NRF5340_XXAA_APPLICATION)
+	*/
+
+#if defined(RADIO_MODE_MODE_Ble_LR500Kbit) || defined(NRF5340_XXAA_APPLICATION)
+static int cmd_phy_coded_s8(const struct shell *shell, size_t argc,
+			    char **argv)
+{
+	test_params.phy->options = BT_CONN_LE_PHY_OPT_CODED_S8;
+	test_params.phy->pref_rx_phy = BT_GAP_LE_PHY_CODED;
+	test_params.phy->pref_tx_phy = BT_GAP_LE_PHY_CODED;
+
+	shell_print(shell, "Preferred PHY set to: %s",
```

Comment:
nit: `PHY set to: ...` for consistency with other prints

Reply:
Ok. Changed